buildpacks/jvm/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Buildpack API version changed from `0.9` to `0.10`, and so requires `lifecycle` `0.17.x` or newer. ([#662](https://github.com/heroku/buildpacks-jvm/pull/662))
 - This buildpack is no longer Heroku stack specific and can be used with most x86 Linux based CNB build and run images. ([#662](https://github.com/heroku/buildpacks-jvm/pull/662))
 - Default OpenJDK distribution is now always Azul® Zulu®. ([#662](https://github.com/heroku/buildpacks-jvm/pull/662))
+- Some error messages relating to OpenJDK installation changed. ([#665](https://github.com/heroku/buildpacks-jvm/pull/665))
+- OpenJDK is now downloaded from `https://s3.amazonaws.com/heroku-buildpacks-jvm`. Users that use allow-listing for internet access during builds might need to add this new URL to their allow-list. ([#665](https://github.com/heroku/buildpacks-jvm/pull/665))
```

Comment:
The `s3.amazonaws.com/<bucket>` style URL is a legacy S3 URL that uses the old global S3 endpoint. This is a regression on the current S3 URL form used by `buildpacks/jvm/src/version.rs` prior to this PR. Could you switch back to the `<bucket>.s3.us-east-1.amazonaws.com` form? See also #314.